dataplane/ebpf/.cargo/config.toml

```diff
@@ -1,6 +1,7 @@
 [build]
 target-dir = "../target"
 target = "bpfel-unknown-none"
+rustflags = ["-C", "save-temps"]
```

Comment:
I think it would be nice to add comments here about our flags for future readers